src/scss/mixins/_bootstrap.scss

```diff
@@ -0,0 +1,100 @@
+@mixin border-radius($radius: $border-radius) {
```

Comment:
Would it maybe be easier to just copy the compiled css for .btn's into the plugin instead of copying down multiple mixin/variable files from Athena?